themes/Solarized-light.json

```diff
@@ -124,7 +124,7 @@
   "colors": {
     "editor.foreground": "#586E75",
     "editor.background": "#FDF6E3",
-    "editor.selectionBackground": "#073642",
+    "editor.selectionBackground": "#EEE8D5",
```

Comment:
Where did you get this color from?

Reply:
Also, these files are autogenerated using tmtheme files. So this may not be the correct way to change color.